spyder_line_profiler/spyder/widgets.py

```diff
@@ -613,23 +771,74 @@ def populate_tree(self):
 
                 # Monospace font for code
                 line_item.setFont(COL_LINE, monospace_font)
+                
+    def on_item_clicked(self, item):
+        data = item.data(COL_POS, Qt.UserRole)
+        if data is None or len(data)<2: return
```

Comment:
Missing spaces and `return` in a new line:
```suggestion
        if data is None or len(data) < 2:
            return
```